packages/docs/webpack.config.js

```diff
@@ -92,6 +84,7 @@ const config = {
     devtool: 'source-map',
 
     devServer: {
+        host: '0.0.0.0',
```

Comment:
The only non-prettier change in this file. Allows dev server to be accessed over a range of different 'local' hostnames, eg. `192.168.1.X`, `10.0.0.X`, `localhost` etc.